demos/multi_channel/common/input.cpp

```diff
@@ -253,23 +256,21 @@ class VideoSourceOCV : public VideoSource {
     std::thread workThread;
     const bool isAsync;
     std::atomic_bool running = {true};
-    std::string videoName;
 
     std::mutex mutex;
     std::condition_variable condVar;
     std::condition_variable hasFrame;
     std::queue<std::pair<bool, cv::Mat>> queue;
 
-    cv::VideoCapture source;
-    bool loopVideo;
+    std::unique_ptr<ImagesCapture> cap;
```

Comment:
Should probably rename this class if it no longer uses OpenCV directly.

Reply:
Renamed